nilearn/masking.py

```diff
@@ -513,6 +513,139 @@ def compute_multi_background_mask(data_imgs, border_size=2, upper_cutoff=0.85,
     return mask
 
 
+def compute_gray_matter_mask(target_img, threshold=1,
+                             connected=True, opening=2,
+                             interpolation='nearest', verbose=0):
+    """ Compute a mask corresponding to the gray matter part of the brain.
+    The gray matter part is found through the resampling of MNI152 template
+    from which the range of values for GM is known
+
+    Parameters
+    ----------
+    target_img: Niimg-like object
+        See http://nilearn.github.io/manipulating_images/input_output.html
+        Images used to compute the mask. 3D and 4D images are accepted.
+        If a 3D image is given, we suggest to use the mean image
```

Comment:
Why is the mean important here? The image is just used for the affine and the bouding box, isn't it. Maye this should be written here.

Reply:
You are right, that was something I wrote in the beginning, I'm fixing it